keras/models.py

```diff
@@ -76,6 +88,49 @@ def standardize_weights(y, sample_weight=None, class_weight=None):
     else:
         return np.ones(y.shape[:-1] + (1,))
 
+def sequential_from_yaml(yamlString):
+    '''
+        Returns a compiled Sequential model generated from a local yaml file,
+        which is either created by hand or from Sequential.to_yaml
+    '''
+    modelYaml = yaml.load(yamlString)
+    model = Sequential()
+
+    class_mode = modelYaml.get('class_mode')
+    theano_mode = modelYaml.get('theano_mode')
+    loss = globals()[modelYaml.get('loss')]
+
+    optim = modelYaml.get('optimizer')
+    optimName = optim.get('name')
+    optim.pop('name')
+    optimizer = globals()[optimName](**optim)
```

Comment:
In general throughout this PR, we should follow PEP8 and avoid camel case. Let's use snake case instead (this is Python after all).